src/main.jsx

```diff
@@ -4,7 +4,7 @@ import App from './App.jsx'
 import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
-  <React.StrictMode>
+  // <React.StrictMode>
```

Comment:
Jajaja, why are you commenting this? Is supposed to find bugs, hide them is cheating 😆